src/tainting/Dataflow_tainting.ml

```diff
@@ -1727,7 +1727,14 @@ let findings_from_arg_updates_at_exit enter_env exit_env : T.finding list =
   (* TOOD: We need to get a map of `lval` to `Taint.arg`, and if an extension
```

Comment:
Could you please create a test case that does not depend on the Flask decorators PR ? And have a separate PR to Pro for it ?

Reply:
done.